src/v2/Apps/ViewingRoom/Routes/Statement/Components/ViewingRoomWorks.tsx

```diff
@@ -30,7 +29,7 @@ const ViewingRoomWorks: React.FC<ViewingRoomWorksProps> = ({
   return (
     <>
       <Flex>
-        {edges.map(({ node: artwork }, index) => {
+        {edges.slice(0, 2).map(({ node: artwork }, index) => {
```

Comment:
For now just limiting this display to 2. Proper fix need to include - displaying first 4 works and converting this to `MasonryGrid` and adding works count to the 'view Works' button.